test/run.d

```diff
@@ -145,7 +181,29 @@ auto predefinedTargets(string[] targets)
         return testPath(dir).dirEntries("*{.d,.sh}", SpanMode.shallow).map!(e => e.name);
     }
 
-    Appender!(string[]) newTargets;
+    static Target createUnitTestTarget()
+    {
+        Target target = {
+            args: [getEnvironment["DMD"], "-run", "unit_test_runner.d"]
+        };
+
+        return target;
+    }
+
+    static Target createDefaultTarget(string filename)
```

Comment:
`default` is a very confusing name.
Maybe sth. like `createTestTarget`?

Reply:
Yeah, perhaps. I couldn't come up with a good name.